assets/app/scripts/directives/logViewer.js

```diff
@@ -71,8 +71,8 @@ angular.module('openshiftConsole')
 
 
 
-
-            var updateScrollLinks = function() {
+            // is just toggling show/hide, nothing else.
+            var toggleScrollLinks = function() {
```

Comment:
Toggle implies you are flipping the state with each call. This function doesn't do that.